app/src/main/java/space/taran/arknavigator/mvp/model/repo/stats/category/TagUsedTSStorage.kt

```diff
@@ -0,0 +1,55 @@
+package space.taran.arknavigator.mvp.model.repo.stats.category
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromStream
+import space.taran.arknavigator.mvp.model.repo.stats.StatsCategory
+import space.taran.arknavigator.mvp.model.repo.stats.StatsEvent
+import space.taran.arknavigator.utils.Tag
+import timber.log.Timber
+import java.nio.file.Path
+import kotlin.io.path.Path
+import kotlin.io.path.inputStream
+import kotlin.io.path.notExists
+import kotlin.io.path.writeText
+
+class TagUsedTSStorage(
```

Comment:
After running it on my device and doing some labeling, I open `.ark/stats/tag-used-ts` and see that there is only 1 tag mentioned. Either timestamp stats are not flushed, or not collected properly.

All looks well with `.ark/stats/tag-labeled-n`.

Reply:
I have now changed the interval to 10 seconds. Please note that the flush occurs if there were no events for 10 seconds. Can you check it again?

Reply:
Let me check again..

Btw, noticed redundant space inside of quotes:
```
{
  "data": {
    " udp": 1666083192624,
    "ark": 1666100238758,
    "password": 1666100240732,
    "poker": 1666356452379,
    "contact": 1666361069198,
    "air": 1666361053895,
    "turkish": 1666356241367,
    "trade": 1666356512390,
    "exam": 1666356535724,
    "pilot": 1666356541726
  }
}
```

Reply:
It looks like tag itself contains space

Reply:
A tag can't contain leading whitespace, it should be trimmed during validation. But it can contain whitespaces inside a tag.